src/sagemaker/jumpstart/enums.py

```diff
@@ -34,6 +34,17 @@ class ModelFramework(str, Enum):
     SKLEARN = "sklearn"
 
 
+class JumpStartModelType(str, Enum):
+    """Enum class for JumpStart model type.
+
+    Open source model refers to JumpStart owned community models.
```

Comment:
Do not use "open source" to refer to JumpStart models. The legal-approved terms are 1) publicly available and 2) proprietary models. If publicly available models is associated with the parameter OPEN_WEIGHTS, I would instead say:
- Publicly available models have open weights and are onboarded and maintained by JumpStart.
- Proprietary models from third-party providers do not have open weights. You must subscribe to proprietary models in AWS Marketplace before use.